### DISCUSSION TITLE
Project custom fields are not shown in Issues page

### DISCUSSION THREAD

**Discussion by shimonelbaz** [community/community#39640] [Projects and Issues]
### Select Topic Area

Bug

### Body

when adding custom fields via the project boards , they are now viewable or searchable via the issues page although they are assigned to the issue in the same repo

**Comment by ossanna16** [comment]
Hi there @shimonelbaz and welcome to our community! Thank you for reporting this bug!

**Reply by dpal** [reply]
I did some research. The project data is stored in separate fields specifically defined for projects. You can find more details on the actual objects and interfaces here:
https://docs.github.com/en/graphql/reference/objects#projectv2
And this is how it can be fetched: https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-api-to-manage-projects#finding-information-about-items-in-a-project
Not very easy because you have to parse the JSON output with jq, which in itself is not a walk in the park.
So making it a part of the standard Issues view would help. Also, being able to search by these fields easily would be nice. I am not sure there is a way to do it.

**Comment by samajammin** [comment]
I'm also interested in this - is there any way to view custom fields on a GH issue? Or are they only accessible in the project board view?
